server/sonar-web/src/main/js/apps/quality-gates/components/Condition.js

```diff
@@ -42,19 +42,19 @@ export default class Condition extends Component {
   componentDidMount() {
     const { condition } = this.props;
 
-    if (!condition.id) {
-      this.refs.operator.focus();
+    // TODO looks like `this.opetator` is always null or undefined
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)